Sources/Plasma/PubUtilLib/plPipeline/pl3DPipeline.h

```diff
@@ -107,6 +107,19 @@ static const float kPerspLayerTrans  = 0.00002f;
 
 static const float kAvTexPoolShrinkThresh = 30.f; // seconds
 
+class plDisplayHelper
+{
+public:
+    virtual plDisplayMode DesktopDisplayMode() = 0;
+    virtual std::vector<plDisplayMode> GetSupportedDisplayModes(hsDisplayHndl display, int ColorDepth = 32) const = 0;
+    
+    static plDisplayHelper* CurrentDisplayHelper() { return fCurrentDisplayHelper; }
+    void MakeCurrentDisplayHelper() { fCurrentDisplayHelper = this; }
```

Comment:
Should probably be `GetInstance()`/`SetInstance()` to match with the rest of the codebase.